docs/stdlib/bool.rst

```diff
@@ -74,8 +72,8 @@ Booleans
         db> select <bool>'True';
         {true}
```

Comment:
Make this the first example and add this as the second example:
> select \<bool\>to_json('true');
> {true}

This way, we'll reflect the content and the order of the new lines 65 and 66 by providing an example of casting a bool first from a string and then from a JSON `true`.